body.tex

```diff
@@ -0,0 +1,440 @@
+   \item distribution generated and published using build-stack Jenkins job
+   \end{itemize}
+\end{itemize}
+
+
+This procedure should be applied for all software products in the DM product tree.
+
+The parameters required are:
+\begin{itemize}
+\item GIT\_REFs: the existing Git references, for example "w.2018.52". This is the stable weekly build identified.
+\item GIT\_TAG: the release candidate to be generated, for example "M.n.p.rc1"
+\end{itemize}
+
+For the Science Pipelines, the above steps are collected in one single Jenkins job:
+
+\begin{verbatim}
+releases/official-release
+\end{verbatim}
+
+Note that for all software product using EUPS, like for example the Science Pipelines, release candidates need to start with a $v$.
+
+\subsection{Other Related Artifacts}
+
+Together with the creation of the release candidate, other artifacts may need to be branched, and possibly have a first release candidate also:
+
+\begin{itemize}
+\item Documentation: to consolidate all information relevant for the new release. This includes the release note.
+As an example, for Science Pipelines, the package \textit{pipelines\_lsst\_io} need to be branched.
+\item  Environment definition packages, like for example packages containing conda environment files (YAML).
+\item  Distribution packages; still referring to the Science Pipelines, the package \textbf{lsst}  where \textit{newinstal.sh} is developed, need to be branched also.
+\item Test data packages.
+\item Other packages not tagged automatically, but required to be inline with the released software.
+\end{itemize}
+
+
+\subsection{Release Candidates Validation} \label{sec:rcvalidation}
+
+The release candidate needs to be validated.
+
+In an ideal case, a test campaign following a specific test plan should be conducted, demonstrating that the release candidate, and therefore the forthcoming release, is behaving as required and expected.
+
+Practically, the validation in many cases can be:
+
+\begin{itemize}
+\item installation/configuration of the binaries packages, or distribution image; usually done manually
+\item inspect of the installation, that all expected files and configuration are there
+\item execution of some demo package available case by case depending of the SW product
+\item try some use cases in order to prove that the release candidate is behaving properly
+\item ask downstream users to act as beta testers.
+\end{itemize}
+
+For Science Pipelines, a characterization report is produced, in order to document the release outputs.
+This is a new document created for each major release.
+
+
+\subsection{Resolving Problems}
+
+In case problems are found during the validation, a DM issue needs to be created in Jira.
+This issue shall follow the usual development process as described in the developer guide.
+The fix will be first implemented in a ticket branch, reviewed and merged to master.
+Once the fix has been proved to work, it can be backported to the release branch.
```

Comment:
Of course, it may be that the problem is discovered in a code that no longer exists in `master` (e.g. because it was deleted after the RC was tagged). In that case, I assume the fix is simply merged to the release branch.

Reply:
This is clarified lather: "In a special case, that an issue cannot be fixed on master, the ticket branch can be opened based on the release branch."